### DISCUSSION TITLE
Offline screens

### DISCUSSION THREAD

**Discussion by j81godoy** [Instawork/hyperview#364] [Q&A] [answered]
Hello,
          can I navigate using local xml screen? A whant to build a app that download all xml's at once on login, than it would load the xml first screen from local database/file and navigate using localstorage xmls. There is any way to that?

**Comment by adamstep** [comment, marked as answer]
Hi @j81godoy, there is not supported in the core Hyperview library. However, if you run a local web server as part of your RN app, then it should be possible to start Hyperview pointed at the localhost URL (`http://localhost:8080/my_file.xml`). I see there's a project to run a local webserver here: https://github.com/futurepress/react-native-static-server However, I have not tried it myself.

**Reply by martin-braun** [reply]
I think it would be great for hyperview to evolve in a way that it has a front-end and a back-end scope for its design origin to allow hybrid approaches. I.e. have some views and their scaffolds locally and provide inner dynamic parts from a remote server.

Also, does hyperview has proper cache controls to fallback to offline mode when the internet cuts? I don't think so, but it would be a great addition to the framework.
